maro/event_buffer/event_buffer.py

```diff
@@ -202,62 +158,38 @@ def execute(self, tick: int) -> List[Event]:
             tick (int): Tick used to process events.
 
         Returns:
-            List[Event]: Pending cascade event list at current point.
+            EventList: A list of events that are pending decisions at the current tick.
```

Comment:
According to the current implementation, ``at the current tick`` is not precise. The return value is the continuous pending decision events?

Reply:
They are decision events at current tick, we will filter the event list at the beginning of execute method to get EventLinkedList of current tick.